test/dune

```diff
@@ -1,7 +1,14 @@
+(executable
+ (name dummy)
+ (public_name dummy)
+ (package obuilder)
+ (modules dummy))
+
 (test
-  (name test)
-  (package obuilder)
-  (deps base.tar)
-  (libraries alcotest-lwt obuilder str logs.fmt))
+ (name test)
+ (package obuilder)
+ (deps base.tar %{bin:dummy})
+ (libraries alcotest-lwt obuilder str logs.fmt)
+ (modules log mock_exec mock_sandbox mock_store test))
```

Comment:
Can we enable windows GitHub Actions tests on this now, at least until we get native Windows support in `ocaml-ci`?

Reply:
I've added a GHA script to build OBuilder on Windows, but running the tests on Windows is an adventure in it's own right, and I'd rather do that on the main PR.

Reply:
Makes sense